script/debugger/debugview.js

```diff
@@ -19,7 +19,6 @@ define('bigscreenplayer/debugger/debugview',
      logBox.style.backgroundColor = '#1D1D1D';
      logBox.style.opacity = 0.9;
      logBox.style.overflow = 'hidden';
-     logBox.style.fontFamily = 'Trebuchet MS';
```

Comment:
Seeing as this replicates the functionality in #21 maybe make this monospace just to make the spacing consistent?

Probably best to close that other PR too